tests/Browser/UserTest.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+namespace Tests\Browser;
+
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Laravel\Dusk\Browser;
+use Tests\DuskTestCase;
+use App\User;
+
+class UserTest extends DuskTestCase
+{
+    /**
+     * Permisson account admin
+     *
+     * @return $admin
+     */
+    public function accountAdmin()
+    {
+        $admin = User::where('email', 'loitq@lifull-tech.vn')->first();
+
+        return $admin;
+    }
+
+    /**
+     * Get data from config to test.
+     *
+     * @return array
+     */
+    public function getDataTest()
+    {
+        return config('constants');
+    }
+
+    /**
+     * Admin logged go to dashboard
+     * case 1-1
+     *
+     * @return void
+     */
+    public function testLoginToPageAdmin()
+    {
+        $admin = $this->accountAdmin();
+        $data = $this->getDataTest();
+
+        if (!$admin) {
+            $admin = factory(User::class)->create([
+                'email' => $data['email']['admin'],
+                'password' => bcrypt('lifull@123'),
+            ]);
+        }
+
+        $this->browse(function (Browser $browser) use ($admin, $data) {
+            $browser->visit($data['path']['login'])
+                ->type('email', $data['email']['admin'])
+                ->type('password', 'lifull@123')
+                ->press('Login')
+                ->assertPathIs($data['path']['admin']);
+        });
+    }
+
+    /**
+     * Admin redirect to page create user
+     * case 1-2
+     *
+     * @return void
+     */
+    public function testClickMenuToPageCreate()
+    {
+        $admin = $this->accountAdmin();
+        $data = $this->getDataTest();
+
+        $this->browse(function (Browser $browser) use ($admin, $data) {
+            $browser->loginAs($admin)
+                ->visit($data['path']['listUser'])
```

Comment:
Why in test case go with whatever on screen but in phpunit go with shortcut url ? Check this issue for all other case too